src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,76 @@
+
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet class responsible for the profile page. */
+public class ProfileServlet extends HttpServlet {
+
+
+/** Store class that gives access to Users. */
+  private UserStore userStore;
+
+
```

Comment:
Maybe too much whitespace? Unsure if this matters, though.

Reply:
We should just be consistent with other servlets. I think in general 1 line for this is appropriate between method definitions in a class